django/cantusdb_project/main_app/views/feast.py

```diff
@@ -1,4 +1,5 @@
 from django.views.generic import DetailView, ListView
+from django.db.models.functions import Lower
```

Comment:
I don't think I would have found this! If it does what I think it does, this is excellent!